src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -80,6 +81,18 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferMode requestedT
             request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/event-stream"));
             var response = await _httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
 
+            try
```

Comment:
Move the `try` up to encompass the `SendAsync` too. That will throw if there's a socket error.